emulsify.php

```diff
@@ -408,6 +408,8 @@ function _emulsify_get_files_to_alter() {
     'emulsify.theme',
     'emulsify.breakpoints.yml',
     'emulsify.libraries.yml',
+    'templates/navigation/menu--main.html.twig',
```

Comment:
It looks like these two lines should be removed since they are already copied by default when the slim option is not used.  See a few lines down where we copy the entire contents of the `templates` and `components` directories.  This would add these files to the `--slim` option which is designed not to copy any files from these directories and be a bare bones install.  Let me know if you were having some trouble with these files not showing up as expected.

Reply:
@ccjjmartin, I was not aware of the `--slim` option.
When I run `php emulsify.php -slim myslimtheme` I am getting an error -
```
PHP Warning:  copy(/home/shaal/code/lando/rosenberg/web/themes/contrib/emulsify/components/css): failed to open stream: No such file or directory in /home/shaal/code/lando/rosenberg/web/themes/contrib/emulsify/emulsify.php on line 125
Failed on Phase: Copy files.Your theme was not successfully created.
```
I think the following line of code is trying to copy a directory that does not exist - `components/css`
https://github.com/fourkitchens/emulsify/blob/develop/emulsify.php#L495

Reply:
@ccjjmartin, I updated the code.
Now it doesn't try copying `components/css` directory
I moved the 2 files that need to be altered to the right place so it works as intended with the `-slim` option or without.
https://github.com/fourkitchens/emulsify/pull/268#commits-pushed-1c572e3